safe_transaction_service/tokens/tasks.py

```diff
@@ -82,7 +82,7 @@ def update_token_info_from_token_list_task() -> int:
     token_addresses = [
         fast_to_checksum_address(token["address"])
         for token in tokens
-        if token["chainId"] == ethereum_network.value
+        if token.get("chainId") == ethereum_network.value
```

Comment:
If there's no `chainId` then we should not include this token at all correct?

Reply:
Yes, I agree, shouldn't be included as we cannot ensure that the token address in the chain. The proposed approach is ok in my opinion because is not including it

Reply:
Makes sense! Thanks 👍